PSReadLine/YankPaste.vi.cs

```diff
@@ -70,6 +70,26 @@ private void SaveLinesToClipboard(int lineIndex, int lineCount)
             _clipboard.LinewiseRecord(_buffer.ToString(range.Offset, range.Count));
         }
 
+        /// <summary>
+        /// Removes a portion of text from the buffer
+        /// and saves it to the clipboard in order to support undo.
+        /// </summary>
+        /// <param name="start"></param>
+        /// <param name="count"></param>
+        /// <param name="instigator"></param>
+        /// <param name="arg"></param>
+        private void RemoveTextToClipboard(int start, int count, Action<ConsoleKeyInfo?, object> instigator = null, object arg = null)
+        {
+            _singleton.SaveToClipboard(start, count);
+            _singleton.SaveEditItem(EditItemDelete.Create(
+                _clipboard,
```

Comment:
Please use `_clipboard.RawText` here to improve the readability. It's not obvious that `ViRegister` implements an implicit converter to string, so it would cost more time to someone who is reading this code to realize that.

Reply:
Thanks for the suggestion.

As a result, and following changes made in this PR, the implicit converter is no longer needed and is therefore removed.